docker/docker.go

```diff
@@ -566,6 +574,75 @@ func (d *ImageManager) RunInContainer(opts RunInContainerOpts) (exitCode int, co
 	}
 	attached <- <-attached
 
+	// Stream files within the container just before starting
+	fsWithSymlinks := fs.NewFileSystem()
+	fsWithSymlinks.KeepSymlinks(true)
+	for src, dst := range opts.StreamIn {
+		tarstream := tarstream.New(fsWithSymlinks)
```

Comment:
Maybe use `tarStream` to avoid using the same name for a package and a variable?